api/v1/model/src/commonMain/kotlin/Repository.kt

```diff
@@ -110,6 +110,5 @@ enum class RepositoryType {
     GIT,
     GIT_REPO,
     MERCURIAL,
-    SUBVERSION,
-    CVS
+    SUBVERSION
```

Comment:
In theory this would also require a database migration to make sure that any existing entries are removed to avoid runtime errors. But I believe in this case the probability that they exist in any production databases is zero because support was removed over a year ago.

Reply:
So, good to merge?